docker-compose.yml

```diff
@@ -53,6 +53,7 @@ services:
     image: *project
     environment:
       << : *default-environment
+      COMPOSER_MEMORY_LIMIT: -1
```

Comment:
this will be applied to all commands running within CLI container. This will not affect `PHP` container.

Reply:
I like this as a solution. I'm just going to do a test on this commit for another project (say `tide_event` as that was the first to give me issues) and will merge if that works.